docs/intro.md

```diff
@@ -24,11 +24,14 @@ docker run -it --rm ghcr.io/cucapra/calyx:latest
 
 The `--rm` flag will remove the container after you exit the shell. If you want to keep the container around, remove the flag.
 
-Just to make sure that all's well with your Docker image, run the core tests:
+Just to make sure that all's well with your Docker image, run the following tests:
 ```
-runt -i core
+runt -i 'core|mrxl'
+fud check verilog jq mrxl interpreter
 ```
 
+If you get any warnings or errors, please report them to us before proceeding.
```

Comment:
This is a bit of a non-generic advice, i.e., it only really applies to the tutorial audience. I suggest we commit this for now and add a post-tutorial cleanup issue to generalize this advice.